_docs/plugins/themes.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: documentation
+title: Themes
+category: Plugins
+order: 5.1
+---
+
+# Finding Themes
+
+Themes published on npm should have the keyword `lounge-theme`, so you can find a list by [searching on npmjs.org](https://www.npmjs.com/search?q=keywords%3Alounge-theme).
+
+# Installing Themes
+
+To install theme `lounge-theme-custom`
+
+* `npm install --prefix $LOUNGE_HOME/packages lounge-theme-custom`
+* Restart The Lounge
+* Select theme in options
+
+# Creating Themes
+
+Create an npm package using the base example.json below, and publish to npm.
+
+```
+{
+  "name": "lounge-theme-custom", // Replace 'custom' with your theme's name
```

Comment:
Comments are not supported in `package.json`, so if people copy-paste as is, that could create confusion.

Also, I don't think we should document about license, issue tracker URL, etc. Just say that themes are normal npm packages, link to npm docs (like [this one](https://docs.npmjs.com/getting-started/creating-node-modules)), and add about the `lounge` property and keywords.
I understand it's a kind thing to do though.

Reply:
Hmmmm alright, how about it just talk about the exports, and mention that if you add "main: package.json" to your package.json, you can have the exports in that. Might be cleaner.